src/jit_compiler_a64.cpp

```diff
@@ -95,6 +95,10 @@ JitCompilerA64::JitCompilerA64()
 {
 	memset(reg_changed_offset, 0, sizeof(reg_changed_offset));
 	memcpy(code, (void*) randomx_program_aarch64, CodeSize);
+
+#ifdef __GNUC__
```

Comment:
Is this really gcc-specific? The Android SDK uses clang, will this work there, or is some other fix needed for an Android build?

Reply:
It's a GCC-specific builtin, but clang supports it and even defines `__GNUC__`. Also, the same `#ifdef` is used in other places here.